view/desktop/bin/src/lib.rs

```diff
@@ -112,6 +112,33 @@ pub fn run() {
                 app.handle().manage(app_state);
             }
 
+            #[cfg(desktop)]
```

Comment:
The application will never run on anything other than a desktop, so I don’t think this clarification is necessary here.